packages/.analysis_options

```diff
@@ -0,0 +1,55 @@
+# Specify analysis options.
```

Comment:
Why is this in `packages/` rather than in the root? We have dart in other subdirectories of this repo.